src/main/java/uk/gov/hmcts/darts/common/repository/UserAccountRepository.java

```diff
@@ -63,4 +63,12 @@ AND securityRole.id IN (:roleIds)
 
     List<UserAccountEntity> findByEmailAddressIgnoreCaseAndActive(String emailAddress, Boolean active);
 
+    @Modifying
+    @Query("""
+        UPDATE UserAccountEntity
+        SET lastLoginTime = CURRENT_TIMESTAMP
```

Comment:
Yep, I'd hope it's fine to use the database function here.
But get what you're saying, it would need to be `OffsetDateTime.now()` really which our application currently enforces UTC as the system time.
But yep, I'd expect that the cloud servers are all running on the same network time anyway (UTC).